src/main/java/org/opentripplanner/standalone/config/ConfigModel.java

```diff
@@ -105,4 +105,15 @@ public static void initializeOtpFeatures(OtpConfig otpConfig) {
     OTPFeature.enableFeatures(otpConfig.otpFeatures);
     OTPFeature.logFeatureSetup();
   }
+
+  /**
+   * Checks if any unknown or invalid properties were encountered while loading the configuration.
+   */
+  public boolean hasInvalidProperties() {
```

Comment:
These are not `InvalidProperties`, they are unused - rename. The main design is to allow for unused properties to exist.

Reply:
bd702cda23caf0b09aa342c932268f955a11469b